test/actions.js

```diff
@@ -96,15 +95,15 @@ describe('NLU', function() {
                     res.body.credentials.should.have.property('url');
                     res.body.credentials.should.have.property('version');
                     res.body.credentials.should.have.property('version_date');
-                    should.notEqual(res.body.credentials.username, 'your wcs username', 'Username is not set.');
-                    should.notEqual(res.body.credentials.password, 'your wcs password', 'Password is not set.');
-                    should.notEqual(res.body.credentials.url, 'your wcs url', 'Url is not set.');
-                    should.equal(res.body.credentials.version, 'v1', 'Should be \'v1\'.');
+                    should.exist(res.body.credentials.username);
```

Comment:
why did you change these checks?

Reply:
In the boilerplate itself the values in the JSON ARE "your wcs url" etc...
So by design it will always fail.
The idea is that once you use the boilerplate this test should remind you to change these values - and then it is no longer a boilerplate.
So wither we don't test it all together or we test something reasnable - I did the later